runhouse/servers/http/http_client.py

```diff
@@ -267,10 +267,14 @@ def check_server(self):
                 f"but local Runhouse version is ({runhouse.__version__})"
             )
 
-    def status(self, resource_address: str):
+    def status(self, resource_address: str, send_to_den: bool = False):
         """Load the remote cluster's status."""
         # Note: Resource address must be specified in order to construct the cluster subtoken
-        return self.request("status", req_type="get", resource_address=resource_address)
+        return self.request(
+            f"status?send_to_den={send_to_den}",
```

Comment:
why do we need to include this param in the request itself made to den?

Reply:
It is the status http request that is sent to the cluster, not to den :)